src/front-end/src/components/Login.js

```diff
@@ -11,9 +12,10 @@ class Login extends Component {
     this.state = {
       email: '',
     };
-
     this.localResetPassword = this.localResetPassword.bind(this);
     this.handleEmailChange = this.handleEmailChange.bind(this);
+    this.Login = this.Login.bind(this);
+    this.Change = this.Change.bind(this);
```

Comment:
The general convention is to write method names in camelCase, let's stick with it.

Reply:
done